server/data/helper_party_api_key.ts

```diff
@@ -8,6 +8,8 @@ export async function validateApiKey(
   key: string,
 ): Promise<boolean> {
   const supabase = await createSupabaseServiceClient();
+  // toISOString() always and only returns UTC timestamps
+  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString
   const currentTimestamp = new Date().toISOString();
```

Comment:
you may want to refactor this later to be able to unit test the expiration

Reply:
could also add an already expired key to seed.sql for that test. it's a good test to add